tests/integration/mirrorpeersecret_controller_test.go

```diff
@@ -601,6 +601,22 @@ var _ = Describe("MirrorPeerSecret Controller", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			err = k8sClient.Delete(ctx, &mirrorPeer)
 			Expect(err).ShouldNot(HaveOccurred())
+		WAIT_LOOP:
```

Comment:
We don't do that here.
Check this: https://github.com/red-hat-storage/ocs-osd-deployer/blob/main/testutils/utils.go#L28-L34